site/docs/install.md

```diff
@@ -67,12 +67,19 @@ package:
 Bazel also comes with a zsh completion script. To install it:
 
 1. Add this script to a directory on your $fpath:
-
-    ```
-    fpath[1,0]=~/.zsh/completion/
-    mkdir -p ~/.zsh/completion/
-    cp scripts/zsh_completion/_bazel ~/.zsh/completion
-    ```
+    1. Linux
+
+        ```
+        fpath[1,0]=~/.zsh/completion/
+        mkdir -p ~/.zsh/completion/
+        cp scripts/zsh_completion/_bazel ~/.zsh/completion
+        ```
+    2. macOS
+
+        ```
+        cp /usr/local/Cellar/bazel/0.5.1/share/zsh/site-functions/_bazel  ./completion/.
```

Comment:
What is ./completion/.?
Why do you need a special step on macOS? Isn't homebrew supposed to install that correctly?

Also why is the step different from Linux? I mean the origin should be the git repository probably.